app/assets/javascripts/spotlight/blocks/resources_block.js

```diff
@@ -51,7 +51,7 @@ Spotlight.Block.Resources = (function(){
                     '<div data-panel-image-pagination="true"></div>',
                   '</div>',
                   '<div class="remove pull-right">',
-                    '<a data-item-grid-panel-remove="true" href="#">Remove</a>',
+                    '<a data-item-grid-panel-remove="true" href="#"><%= i18n.t("blocks:resources:panel:remove") %></a>',
```

Comment:
It don't think this is using the (lodash? underscore?) templates so these are just getting printed out as strings:

### Remove link
![remove](https://cloud.githubusercontent.com/assets/96776/6647099/54f06d88-c988-11e4-8402-7b2cfaa5fc4a.png)

### Display? label
![display-label](https://cloud.githubusercontent.com/assets/96776/6647098/54ef2cf2-c988-11e4-84c8-8fea31d3e309.png)